src/vm-memory/Cargo.toml

```
@@ -0,0 +1,11 @@
+[package]
+name = "vm-memory"
+version = "0.1.0"
+authors = ["Amazon Firecracker team <firecracker-devel@amazon.com>"]
+edition = "2018"
+
+[dependencies]
+vm-memory = { version = ">=0.2.2", features = ["backend-mmap"] }
```

This
```suggestion
vm-memory-upstream = { package = "vm-memory", version = ">=0.2.2", features = ["backend-mmap"] }
```
and replacing occurrences of `vm_memory` with `vm_memory_upstream` inside the `firecracker/vm_memory` crate should do the trick for the doctests

---

Thanks, this fixes the doctest issue.